lightning/src/ln/channelmanager.rs

```diff
@@ -1203,6 +1204,37 @@ impl ChannelDetails {
 	}
 }
 
+/// Used to express the status of payments that are currently pending resolution. They do not
```

Comment:
Let's link to `ChannelManager::list_pending_payments`.

Reply:
Resolved in 61d4e31b